src/Symfony/Component/Config/Tests/FileLocatorTest.php

```diff
@@ -86,7 +87,7 @@ public function testLocate()
     }
 
     /**
-     * @expectedException \InvalidArgumentException
+     * @expectedException FileLocatorFileNotFoundException
```

Comment:
I have updated the PR accordingly. However, `use` statements seem to work in phpunit under PHP 5.6 and PHP 7. Is this a known issue?